src/inference/src/dev/core_impl.cpp

```diff
@@ -1430,6 +1431,12 @@ void ov::CoreImpl::add_extensions_unsafe(const std::vector<ov::Extension::Ptr>&
     }
 }
 
+std::vector<ov::Extension::Ptr> ov::CoreImpl::get_extensions_copy() const {
+    std::lock_guard<std::mutex> lock(get_mutex());
+    std::vector<ov::Extension::Ptr> extensions_copy(extensions.begin(), extensions.end());
+    return extensions_copy;
```

Comment:
Coverity might complain.
Consider const iterators.

```suggestion
    return {extensions.cbegin(), extensions.cend()};
```

Reply:
RVO takes place in such case and Coverity should handle - comment invalid.